src/gui/editors/PianoRoll.cpp

```diff
@@ -738,6 +741,10 @@ void PianoRoll::setCurrentPattern( Pattern* newPattern )
 	connect( m_pattern->instrumentTrack(), SIGNAL( midiNoteOff( const Note& ) ), this, SLOT( finishRecordNote( const Note& ) ) );
 	connect( m_pattern->instrumentTrack()->pianoModel(), SIGNAL( dataChanged() ), this, SLOT( update() ) );
 
+	connect(m_pattern->instrumentTrack()->microtuner()->keymapModel(), SIGNAL(dataChanged()), this, SLOT(update()));
+	connect(m_pattern->instrumentTrack()->microtuner()->keyRangeImportModel(), SIGNAL(dataChanged()),
+				this, SLOT(update()));
```

Comment:
Let's not try to align with just tabs. If we have to split the line, personally I like the Python way:
```suggestion
	connect(
		m_pattern->instrumentTrack()->microtuner()->keyRangeImportModel(),
		SIGNAL(dataChanged()),
		this,
		SLOT(update())
	);
```

Reply:
Yeah, I now prefer the second style instead of using tabs for alignment as well, but half a year ago when I wrote most of the stuff I had a different preference. :) So there may be more of these all over the place.

Reply:
No worries! I think we'd all prefer readability over long lines. :D